src/main/java/bridge/model/BridgeSize.java

```diff
@@ -0,0 +1,29 @@
+package bridge.model;
+
+import bridge.utils.enums.ErrorMessage;
+
+import static bridge.utils.enums.Const.BRIDGE_MAX_SIZE;
+import static bridge.utils.enums.Const.BRIDGE_MIN_SIZE;
+
+public class BridgeSize {
+    private final int value;
+
+    public BridgeSize(int value) {
+        validate(value);
+        this.value = value;
+    }
+
+    public int getValue() {
+        return value;
+    }
+
+    private void validate(int value) {
+        validateNumberRange(value);
+    }
+
+    private void validateNumberRange(int value) {
+        if (value < BRIDGE_MIN_SIZE.getNumber() || value > BRIDGE_MAX_SIZE.getNumber()) {
+            throw new IllegalArgumentException(ErrorMessage.INVALID_RANGE.getMessage());
+        }
+    }
```

Comment:
저도 모두 지키진 못하지만 이미 아실 내용일 수도 있지만 귀찮아서 아래에 구현하셨을 수도 있다고 생각하는데 이미 아시는 내용이더라도 리마인드하시라고 클린 코드 책 관련해서 내용 전달드립니다

클린코드 5장에서 코드 형식에 관한 이야기가 나올 때 수직 거리에 대해서 이야기가 나옵니다.

"한 함수가 다른 함수를 호출한다면 가능하다면 호출하는 함수를 호출되는 함수보다 먼저 배치한다.",
"개념적인 친화도가 높을수록 코드를 가까이 배치한다." 
위와 같은 이야기들이 존재하니 그냥 지나가면서 참고만 하시면 좋을 것 같습니다 !
```suggestion
    private void validate(int value) {
        validateNumberRange(value);
    }

    private void validateNumberRange(int value) {
        if (value < BRIDGE_MIN_SIZE.getNumber() || value > BRIDGE_MAX_SIZE.getNumber()) {
            throw new IllegalArgumentException(ErrorMessage.INVALID_RANGE.getMessage());
        }
    }
    
    public int getValue() {
        return value;
    }
```

 저도 클린코드를 지키려고 노력은 하지만 쉽게는 안되네요 ㅠㅠ 코드상에 문제는 없습니다 !